third_party/brotli/c/common/platform.c

```diff
@@ -4,7 +4,11 @@
    See file LICENSE for detail or copy at https://opensource.org/licenses/MIT
 */
 
-#include <stdlib.h>
```

Comment:
@y4vor do we still need this ? E.g. does something break if we include stdlib ?

Reply:
Just to note this file (platform.c) does not exist at all on the main branch:
github.com/youtube/cobalt/tree/main/third_party/brotli/c/common

Reply:
We can try to bring it back. If it builds and there is no API leak it is fine.

Reply:
I run the internal build, and there is no API leakage. (See my CL in gerrit.) But in this file, I see malloc is being used. @y4vor Could you confirm I'm doing it right?

Reply:
Discussed this with @y4vor offline. So maybe the code is not being used in Cobalt, so no API leakage introduced. We can just use this standard library to keep the minimum change in third-party library.